src/main/java/racingcar/controller/RacingGameController.java

```diff
@@ -0,0 +1,28 @@
+package racingcar.controller;
+
+import racingcar.domain.RacingGame;
+import racingcar.view.InputView;
+import racingcar.view.ResultView;
+
+import java.util.InputMismatchException;
+
+public class RacingGameController {
+
+    public void RacingGameStart() {
+        try {
+            InputView inputView = new InputView();
+            String cars = inputView.getInputCarNames();
+            int time = inputView.getInputTime();
+
+            RacingGame racingGame = new RacingGame(cars, time);
+            racingGame.start();
+
+            ResultView resultView = new ResultView(racingGame);
+            resultView.printRacingGameWinner();
+
+        } catch (
+        InputMismatchException e) {
```

Comment:
아마 단순 오타인 것으로 보이긴 하지만, 혹시나 해서요. 좌괄호 오픈하시면서 파라미터도 같은 줄에 써 주시고요. 

```suggestion
    } catch(InputMismatchException e) {